app/tests/spec/views/confirm.js

```diff
@@ -363,7 +352,39 @@ define(function (require, exports, module) {
       it('triggers gmail tab opening', function () {
         sinon.spy(view.notifier, 'trigger');
         view._gmailTabOpened();
-        assert.isTrue(view.notifier.trigger.called);
+        assert.isTrue(TestHelpers.isEventLogged(metrics, 'confirm.openGmail.clicked'));
+      });
+    });
+
+    describe('openGmail feature', function () {
+      it('it is not visible in basic contexts', function () {
+        assert.notOk($('#open-gmail').length);
+      });
+
+      it('it is visible in desktop contexts', function () {
+        broker = new FxDesktopV3AuthenticationBroker({
+          relier: relier,
+          session: Session,
+          window: windowMock
+        });
+
+        view = new View({
+          broker: broker,
+          fxaClient: fxaClient,
+          metrics: metrics,
+          model: model,
+          notifier: notifier,
+          relier: relier,
+          user: user,
+          viewName: 'confirm',
+          window: windowMock
+        });
+
+        return view.render()
+          .then(function () {
+            $('#container').html(view.el);
+            assert.ok($('#open-gmail').length);
```

Comment:
You can do an `assert.lengthOf(view.$('#open-gmail'), 1)` - I just learned about Chai's [assert.lengthOf](http://chaijs.com/api/assert/#lengthOf) test recently.